packages/react-components/src/components/VideoTile.tsx

```diff
@@ -189,18 +187,24 @@ const videoTileMoreMenuProps = {
   styles: { container: { maxWidth: '8rem' } }
 };
 /* @conditional-compile-remove(pinned-participants) */
-const VideoTileMoreOptionsButton = (props: { contextualMenu?: IContextualMenuProps }): JSX.Element => {
-  const { contextualMenu } = props;
+const VideoTileMoreOptionsButton = (props: {
+  contextualMenu?: IContextualMenuProps;
+  canShowContextMenuButton: boolean;
+}): JSX.Element => {
+  const { contextualMenu, canShowContextMenuButton } = props;
   if (!contextualMenu) {
     return <></>;
   }
+
+  const optionsIcon = canShowContextMenuButton ? 'VideoTileMoreOptions' : 'VideoTileHideOptionsIcon';
```

Comment:
What is the purpose behind the `VideoTileHideOptionsIcon` ? Are we unable to not render the `<IconButton` when `canShowContextMenuButton` is false?

Reply:
It renders the icon button without any icon, so the button is there and it can be navigated using keyboard to activate the context menu. We use similar solution for participant Item where the ellipsis appear on hover.